src/app/containers/ProductDetailsPage/ProductDetailsPage.js

```diff
@@ -1,31 +1,31 @@
-import React, {Component} from 'react';
+// import React, { Component } from 'react';
 
-class ProductDetailsPage extends Component {
-  constructor(props) {
-    super(props);
-    const { match: { params } } = this.props;
+// class ProductDetailsPage extends Component {
+//   constructor(props) {
+//     super(props);
+//     const { match: { params } } = this.props;
 
-    this.state = {
-      product: {
-        name: params.name,
-        id: params.id
-      }
-    }
-  }
+//     this.state = {
+//       product: {
+//         name: params.name,
+//         id: params.id,
+//       },
+//     };
+//   }
 
-  returnBack = () => {
-    this.props.history.goBack();
-  }
-  
-  render() {
-    return(
-      <div>
-        <div>{this.state.product.name}</div>
-        <div>{this.state.product.id}</div>
-        <button onClick={() => this.returnBack()}>Go Back</button>
-      </div>
-    );
-  }
-}
+//   returnBack = () => {
+//     this.props.history.goBack();
+//   }
 
-export default ProductDetailsPage;
\ No newline at end of file
+//   render() {
+//     return (
+//       <div>
+//         <div>{this.state.product.name}</div>
+//         <div>{this.state.product.id}</div>
+//         <button onClick={() => this.returnBack()}>Go Back</button>
+//       </div>
+//     );
+//   }
+// }
+
+// export default ProductDetailsPage;
```

Comment:
I don't get why this file was commented out?

Reply:
our config requires propTypes now, i can't push without it